### PR TITLE
Adding Windows troubleshooting section

### PR DESCRIPTION
**Pull request by PatrickLang**
**What this PR does / why we need it**:
We're changing too much in the Windows setup recently so we should doc what works, doesn't, and troubleshooting steps.

**If applicable**:
- [x] documentation

### CODE REVIEW

**Comment by PatrickLang** [comment]
/do-not-merge
Waiting on reviews from people who know the network status better than I do :)

**Comment by PatrickLang** [comment]
This includes info on current issues:
https://github.com/Azure/azure-container-networking/issues/146
https://github.com/Azure/azure-container-networking/issues/147
https://github.com/Azure/acs-engine/issues/3037

**Comment by PatrickLang** [comment]
@daschott @jmesser81 or @dineshgovindasamy can you help review for accuracy?

File: docs/kubernetes/windows.md [hunk 1]
```diff
@@ -297,7 +297,127 @@ spec:
...
+2.	Instead of DNS, use environment variables `* _SERVICE_HOST` and `*_SERVICE_PORT` to find service IPs and ports in the same namespace
+
+
+#### Pods cannot ping default route or internet IPs
+
+Affects: All acs-engine deployed clusters
+
+ICMP traffic is not routed between private Azure vNETs or to the internet.
+
+Workaround: test network connections with another protocol (TCP/UDP)
```
Comment (daschott) [inline docs/kubernetes/windows.md]: Should we clarify how? Like give an example such as:
The following will *not* work to demonstrate connectivity:
`ping 123.456.78`
Instead, you should use something like:
use `curl -Uri http://123.456.78 -UseBasicParsing`

**Pull request merged**